cg/meta/workflow/nextflow_common.py

```diff
@@ -31,9 +31,20 @@ def verify_case_config_file_exists(cls, case_id: str, root_dir: str) -> None:
             raise ValueError(f"No config file found for case {case_id}")
 
     @classmethod
-    def get_case_config_path(cls, case_id: str, root_dir: str) -> Path:
+    def get_case_config_path(cls, case_id: str, root_dir: str) -> str:
         """Generates a path where the Rnafusion sample sheet for the case_id should be located."""
-        return Path((cls.get_case_path(case_id, root_dir)), f"{case_id}_samplesheet.csv")
+        return Path((cls.get_case_path(case_id, root_dir)), f"{case_id}_samplesheet.csv").as_posix()
+
+    @classmethod
+    def get_params_file_path(
+        cls, case_id: str, root_dir: str, params_file: Optional[str] = None
+    ) -> str:
+        """Generates a path where the Rnafusion default params_file a case_id should be located."""
+        if params_file:
+            return params_file
+        return Path(
+            (cls.get_case_path(case_id, root_dir)), f"{case_id}_params_file.yaml"
+        ).as_posix()
```

Comment:
I wonder if it not would be better to return a pathlib object until you actually need to convert to str, i.e handle that on the caller side when needed (like writing to file)